src/index.css

```diff
@@ -14,6 +14,13 @@ code {
 
 /* buttons */
 
+.layer-buttons {
```

Comment:
In general, I'd prefer if we had CSS rules imported only into the components that use them instead of at the app level. I think one approach to this is called "CSS modules" and may require some extra webpack config. 
https://flaviocopes.com/react-css/

I'm also open to using CSS-in-JS libraries like [Emotion](https://emotion.sh/docs/introduction) or [Styled Components](https://www.styled-components.com/) or whatever the new cool thing is. I just don't haven't played around with any of these yet. This seems like a nice small project to experiment with.

I say this, but looking back at the mobile signals work orders app I'm seeing my pretty weak CSS patterns in place. So open to any thoughts or ideas here. I've used Sass preprocessor with [BEM](http://getbem.com/) naming conventions a bunch in the past and have really enjoyed that, but I'm not sure how relevant that is these days...

articles: 
- https://blog.bitsrc.io/5-ways-to-style-react-components-in-2019-30f1ccc2b5b
- https://survivejs.com/react/advanced-techniques/styling-react/

Reply:
I really like the idea of CSS-in-JS so I went with Styled Components since I haven't had a chance to explore it. I changed the LayerButtons component styles and plan to use this library to clean up the CSS in this project.